packages/editor-ui/src/views/SetupWorkflowFromTemplateView/SetupWorkflowFromTemplateView.vue

```diff
@@ -84,7 +85,16 @@ setupTemplateStore.setTemplateId(templateId.value);
 
 onMounted(async () => {
 	await setupTemplateStore.init();
-	await skipIfTemplateHasNoCreds();
+	const wasSkipped = await skipIfTemplateHasNoCreds();
+	if (!wasSkipped) {
+		$telemetry.track(
+			'User opened cred setup',
+			{},
```

Comment:
Nit: Do we need to send an empty object here? Can we just skip the param?

Reply:
Valid point. Changed it to undefined